packages/api/src/command/medical/document/document-query.ts

```diff
@@ -116,7 +115,7 @@ export async function queryDocumentsAcrossHIEs({
 
   const commonwellEnabled = await isCommonwellEnabled();
   if (!cqManagingOrgName) {
-    if (commonwellEnabled || forceCommonwell || Config.isSandbox()) {
```

Comment:
`commonwellEnabled` I believe is true in the sandbox app config

```
  "commonwellFeatureFlag": {
    "enabled": true
  },
```

https://us-west-2.console.aws.amazon.com/systems-manager/appconfig/applications/h3ws9h8/configurationprofiles/7yz6xfu/versions?region=us-west-2&rev1=24